src/dist/conf/config.yaml

```diff
@@ -23,6 +23,8 @@ omero.server:
     # OMERO_HOME/lib/scripts
     omero.script_repo_root: "/opt/omero/lib/scripts"
     omero.pixeldata.max_tile_length: "2048"
+    omero.host: "localhost" #For verticle bean instantiation
```

Comment:
This duplicates this configuration and is not needed as the verticles can pick up this configuration from the Vertx configuration available from `config()`.